src/Deposit.js

```diff
@@ -811,52 +880,75 @@ export default class Deposit {
     }
     /** @type {AutoSubmitState} */
     const state = (this.autoSubmittingState = {})
+    state.fundingTransaction = this.fundingTransaction
+    state.fundingConfirmations = this.fundingConfirmations
```

Comment:
I wonder if we should include these in the state at all given they're now exposed directly on the deposit… My original thinking I believe was that we not lock the deposit into a single minting/confirmation/etc call, thus loading it all into this state object, but I kind of prefer the setup in this PR.

Dropping it from the auto-submit state can wait, though, haven't thought through all the implications yet.

Reply:
> My original thinking I believe was that we not lock the deposit into a single minting/confirmation/etc call, thus loading it all into this state object, but I kind of prefer the setup in this PR.

How about intro'ing a `fundingConfirmationsFor` with the transaction as a parameter eg.

```js
// manual usage
const transaction = await deposit.fundingTransaction
await deposit.fundingConfirmationsFor(transaction)

// in Deposit
get fundingConfirmations() {
    return this.fundingTransaction.then(transaction => this.fundingConfirmationsFor(transaction))```
}